personal_assistant/main.py

```diff
@@ -78,6 +78,38 @@ def main():
         elif command == "entries":
             print(num_records(personal_assistant))
 
+        elif command == "add-note":
+            personal_assistant.add_note()
+
+        elif command == "remove-note":
+            note_name = input("Enter the name of the note to delete: ")
+            personal_assistant.remove_note(note_name)
+            print("Note deleted successfully.")
+
+        elif command == "show-notes":
+            all_notes = personal_assistant.get_all_notes()
+            if all_notes:
+                print("All Notes:")
+                for note in all_notes:
+                    print(note)
```

Comment:
![image](https://github.com/nazarsmith/ppap7856-pythoneers-brigade-prjct/assets/61598939/2f85fbf2-3788-4747-8b7f-1054fc7b1dc0)
maybe we would benefit from a bit of formatting? For example:
##  |                     Name                  |                   Tags                   |                                 Text
or something similar? What do you think?